DATA/production/configurations/2022/LHC22f/apass1_TPCcalib/setenv_extra.sh

```diff
@@ -143,4 +143,5 @@ export CALIB_PHS_ENERGYCALIB=0
 export CALIB_PHS_BADMAPCALIB=0
 export CALIB_PHS_TURNONCALIB=0
 export CALIB_PHS_RUNBYRUNCALIB=0
+export CALIB_TPC_VDRIFTTGL=0
```

Comment:
Here you are disabling it, while the PR title is "enable": what did you mean to do?

Reply:
Also note that enabling a calibration in the processing does not mean that we will extract it, because no aggregation is currently done. If you need to store an output file, let us know, we need to add it to the JDL.

Reply:
PR enables it generally in the setenv_calib. Since `apass1_TPCcalib/setenv_extra.sh` disables all calibrations I've disabled also drift calib there. This calibration is for the online processing.

Reply:
Ok, now I understand the PR title. Thanks! Merging.